libs/fluxc-plugin/src/main/kotlin/org/wordpress/android/fluxc/network/rest/wpcom/wc/bookings/BookingsRestClient.kt

```diff
@@ -131,4 +131,7 @@ private val BookingUpdatePayload.asMap: Map<String, Any>
         note?.let {
             put("note", it)
         }
+        status?.let {
+            put("status", it.key)
+        }
     }
```

Comment:
Minor suggestion you can use `putIfNotNull` to simplify this code: 


```suggestion
private val BookingUpdatePayload.asMap: Map<String, Any>
    get() = mutableMapOf<String, Any>().putIfNotNull(
        "attendance_status" to attendanceStatus?.key,
        "note" to note,
        "status" to status?.key
    )
```

Reply:
TIL! Love it, updated!